zephyr/panic.c

```diff
@@ -1,9 +1,13 @@
 // SPDX-License-Identifier: BSD-3-Clause
 //
 // Copyright(c) 2019 Intel Corporation. All rights reserved.
-//
-// Author: Tomasz Lauda <tomasz.lauda@linux.intel.com>
 
 #include <sof/panic.h>
+#include <sof/trace.h>
+
+void __panic(uint32_t p, char *filename, uint32_t linenum)
+{
+	trace_error(TRACE_CLASS_MEM, "%s : %u", filename, linenum);
```

Comment:
I'm not sure TRACE_CLASS_MEM is the best class for this trace. When I did my own panic code I used TRACE_CLASS_IPC, maybe that one is better? Besides that, LGTM.

Reply:
Oh, also, traces don't work properly with %s.